Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -29,7 +29,8 @@ class PadResponse;
 class Digitizer
 {
  public:
-  enum { kTimeBins = 30 };
+  enum { kTimeBins = 30,
+         kNdet = 540 };
```

Comment:
I think here I would not use enums but a simple `constexpr int`. Also could you apply this systematically everywhere (also in Digitizer.cxx)?

Reply:
Here, `kNdet` was already defined in `TRDCommonParams`. I think it is ok to move there also `kTimeBins`, but the cost is including the `CommonParams` header file in `Digitizer.h`. This might be a good idea since `kTimeBins` should be used across other places too.  If you are ok with these changes, I will propagate this behavior over the next developments.

Reply:
There are still many hardcoded `540` in the code. Could you not use the constant properly right now?

Reply:
Yes, of course. Those numbers were overlooked.